src/fabric/src/fabric.erl

```diff
@@ -525,7 +525,8 @@ inactive_index_files(DbName) ->
 
     if ActiveSigs =:= [] -> FileList; true ->
         lists:filter(fun(FilePath) ->
-            not maps:is_key(filename:basename(FilePath, ".view"), ActiveSigs)
+            not maps:is_key(filename:basename(
+                filename:basename(FilePath, ".view"), ".compact"), ActiveSigs)
```

Comment:
this doesn't seem right. You check for `<sig>.view.compact`. However view compaction files are named `<sig>.compact.view` see [here](https://github.com/apache/couchdb/blob/master/src/couch_mrview/src/couch_mrview_util.erl#L834).

Reply:
No. What I check is `<sig>. compact.view`.

For `sig1.view`, the result of `filename:basename(filename:basename(FilePath, ".view"), ".compact")` is sig1.

For `sig2.compact.view`, the result of `filename:basename(filename:basename(FilePath, ".view"), ".compact")` is sig2.

So all view files and ongoing compact files are considered.

Reply:
Indeed:
```
Eshell V9.3.3.6  (abort with ^G)
1> FilePath = "sig2.compact.view".
"sig2.compact.view"
2> filename:basename(filename:basename(FilePath, ".view"), ".compact").
"sig2"
3> f(FilePath), FilePath = "sig2.view".
"sig2.view"
4> filename:basename(filename:basename(FilePath, ".view"), ".compact").
"sig2"
```

Reply:
I think it would be easier to understand if we would abstract the nested filebase operations into a separate function and add comment there which mention both possible filename formats. The function could be called `get_view_signature_from_filename` or something shorter.